runtime/lua/vim/lsp.lua

```diff
@@ -830,7 +830,7 @@ function lsp.start_client(config)
     rpc.request('initialize', initialize_params, function(init_err, result)
       assert(not init_err, tostring(init_err))
       assert(result, "server sent empty result")
-      rpc.notify('initialized', {[vim.type_idx]=vim.types.dictionary})
+      rpc.notify('initialized', {})
```

Comment:
Shouldn't this use `vim.empty_dict()`?

https://microsoft.github.io/language-server-protocol/specification#initialized

Reply:
nevermind, ignore me, anything satisfies an empty interface 🙈

Reply:
I think it changes the gopls behavior. If I change it as `vim.empty_dict()` the gopls works as the same as the previous version.

Reply:
Nice! Thanks for spotting this, I assumed sending an empty lua table would be fine here.

Reply:
Fixed.